indexer/workers/importer.py

```diff
@@ -153,15 +152,21 @@ def index_routing(self, publication_date_str: Optional[str]) -> str:
         year = -1
         if publication_date_str:
             try:
-                year = datetime.strptime(publication_date_str, "%Y-%m-%d").year
+                pub_date = datetime.strptime(publication_date_str, "%Y-%m-%d")
+                year = pub_date.year
+                # check for exceptions of future dates just in case gets past mcmetadata
+                if pub_date > datetime.now() + timedelta(days=+90):
```

Comment:
```suggestion
                if pub_date > datetime.now() + timedelta(days=90):
```
Why the `+`?

... and that `90` should probably be a constant declared somewhere.